CHANGELOG.md

```diff
@@ -9,7 +14,7 @@
 #### [v0.9.3](https://github.com/heptio/ark/releases/tag/v0.9.3) - 2018-08-10
 
 #### Bug Fixes
-  * Initalize Prometheus metrics when creating a new schedule (#689, @lemaral)
+  * Initalize Prometheus metrics when creating a new schedule (#689, @lemaral) 
```

Comment:
Was this line just changed for whitespace formatting? I notice it has an extra space at the end.

Reply:
repushed